src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/BackupConfiguration.kt

```diff
@@ -0,0 +1,46 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import com.atlassian.performance.tools.jiraactions.Patience
+import org.apache.logging.log4j.LogManager
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+
+class BackupConfiguration(
+    private val driver: WebDriver,
+    private val access: AdminAccess
+) {
+    private val logger = LogManager.getLogger(this::class.java)
+    private val backupLocator = By.id("del_10001")
```

Comment:
Looks like a generated and data specific id. Is it always the same on all Jira instances over the world?

Reply:
It does seem weirdly specific. If we can find a more stable-looking selectors, let's use that.
Otherwise, let's use this one. If we detect bugs with this, we'll react and fix, using the bug details as guidance.